src/MetadataManager/EventManager.ts

```diff
@@ -272,11 +274,10 @@ export class MetadataEventManager {
         const keyResult=keyList.filter(key=> !metadata[key]["is-deleted"]); // find activate. or last key of KeyList;
 
         //data copy
-        let data=JSON.parse(JSON.stringify(metadata[keyList[keyList.length-1]]));
-        if(keyResult.length){data=JSON.parse(JSON.stringify(metadata[keyResult[0]]));}
-
-        else {data["is-deleted"]=false;}
 
+        let data=metadata[keyList[keyList.length-1]];
+        if(keyResult.length){ data=metadata[keyResult[0]]; }
+        else {data["isDeleted"]=false;}
```

Comment:
As you know, we decided that metadata information keys are named by kebab-case today.